tsl/test/sql/include/skip_scan_query_ht.sql

```diff
@@ -0,0 +1,24 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+
+CREATE INDEX ON :TABLE(dev);
+CREATE INDEX ON :TABLE(time);
```

Comment:
We should also add a test for skipscan with index on character datatypes.

Reply:
there are tests for text column (dev_name)